joomla/40-backend-template
# Refactor sendtestmail.js

Pull Request for Issue # .

Note: this needs a PR from @andrepereiradasilva which is not yet merged, so don't merge it yet
### Summary of Changes
- use Joomla.request(), vanilla
### Testing Instructions

Check that the button send test mail in the configuration still works.
(right now you need to check the XHR response in the browser as the messages are not displaying)
### Documentation Changes Required

None, simple refactor
